backend/db/migrations/V1.0.24__add_reports_views.sql

```diff
@@ -0,0 +1,103 @@
+SET search_path TO pay_transparency;
+
+
+CREATE VIEW reports_view
```

Comment:
it should always be Create or Replace follow the existing pattern.
I don't understand why do we need separate .sql files after you have created a view. This look more complicated than needed.

Reply:
We need to add a migration to create the view, what other way do you suggest it can be done. I thought we are following the normal db migration changes. I think we should talk about it first

Reply:
I created a migration file to add the 2 views, and then did a `db pull`, resulting in the schema file  update and 2 other SQL files generated in the prisma/views directory.